Vizzuality/half-earth-v3
# Epic/national report cards

## National Report Cards
### Description
This PR adds the National Report Cards to the `data globe`.
On this new view mode the users can select a country to focus the attention on. This is the first iteration of this feature.
### Designs
[Global scene](https://projects.invisionapp.com/d/main#/console/17484384/409039747/preview)
[Local scene](https://projects.invisionapp.com/d/main#/console/17484384/409039745/preview)
### Testing instructions
Check a complete user flow on the data globe activating and deactivating the data layers and testing that the landscape view mode is working properly.
Display layers metadata modal by clicking on the `info` button displayed on layers selectors after toggling on a given layer. Also check the metadata modal by activating it through the legend items.
Visit the country view and activate and deactivate layers.
### Feature relevant tickets
Epic stories are available on the [National Report Cards](https://www.pivotaltracker.com/epic/show/4528630) pivotal ticket
